src/cc65/staticassert.h

```diff
@@ -0,0 +1,51 @@
+/*****************************************************************************/
+/*                                                                           */
+/*                               staticassert.h                              */
+/*                                                                           */
+/*          _Static_assert handling for the cc65 C compiler                  */
+/*                                                                           */
+/*                                                                           */
+/*                                                                           */
+/* Copyright 2020 Google LLC                                                 */
+/*                                                                           */
+/*                                                                           */
+/* This software is provided 'as-is', without any expressed or implied       */
+/* warranty.  In no event will the authors be held liable for any damages    */
+/* arising from the use of this software.                                    */
+/*                                                                           */
+/* Permission is granted to anyone to use this software for any purpose,     */
+/* including commercial applications, and to alter it and redistribute it    */
+/* freely, subject to the following restrictions:                            */
+/*                                                                           */
+/* 1. The origin of this software must not be misrepresented; you must not   */
+/*    claim that you wrote the original software. If you use this software   */
+/*    in a product, an acknowledgment in the product documentation would be  */
+/*    appreciated but is not required.                                       */
+/* 2. Altered source versions must be plainly marked as such, and must not   */
+/*    be misrepresented as being the original software.                      */
+/* 3. This notice may not be removed or altered from any source              */
+/*    distribution.                                                          */
+/*                                                                           */
+/*****************************************************************************/
```

Comment:
Maybe I'm getting this wrong, but at least as far as I understand we can't have a source file with such a copyright notice in the cc65 code base.